### PR TITLE
Add support for page images with absolute URLs

### PR DESCRIPTION
**Pull request by pathawks**

### CODE REVIEW

**Comment by benbalter** [comment]
A feature that absorbs complexity for users, accounts for them not reading the manual, and does so without adding a setting?!?! 💖

File: lib/template.html [hunk 1]
```diff
@@ -85,11 +85,19 @@
...
+  {% unless seo_site_logo contains "://" %}
+    {% assign seo_site_logo = seo_site_logo | prepend: seo_url %}
+  {% endunless %}
+  {% assign seo_site_logo = seo_site_logo | escape %}
 {% endif %}
 
 {% if page.image %}
-    {% assign seo_page_image = page.image | prepend: seo_url | escape %}
+  {% assign seo_page_image = page.image %}
+  {% unless seo_page_image contains "://" %}
```
Comment (DirtyF) [inline lib/template.html]: contains "//" in case of prefix-free URLs like  `//jekyllrb.com/img/logo-2x.png` ?

Comment (pathawks) [inline lib/template.html]: http://www.paulirish.com/2010/the-protocol-relative-url/

> Now that SSL is encouraged for everyone and doesn’t have performance concerns, *this technique is now an anti-pattern.* If the asset you need is available on SSL, then *always* use the `https://` asset.

**Comment by pathawks** [comment]
I am sure this conflicts with #78.

**Comment by benbalter** [comment]
@pathawks I just merged in https://github.com/jekyll/jekyll-seo-tag/pull/78 to get out of your way. Looking at the diff, I *believe* it should just be a one line conflict. But feel free to clobber my changes as you see fit, and merge when you're ready. Once you do, glad to cut a minor release.

**Comment by pathawks** [comment]
Go team 🎉 

@jekyllbot: merge

**Comment by pathawks** [comment]
@jekyllbot: merge

File: lib/template.html [hunk 2]
```diff
@@ -85,11 +85,20 @@
...
+  {% unless seo_site_logo contains "://" %}
+    {% assign seo_site_logo = seo_site_logo | prepend: seo_url %}
+  {% endunless %}
+  {% assign seo_site_logo = seo_site_logo | escape %}
 {% endif %}
 
 {% if page.image %}
-    {% assign seo_page_image = page.image.path | default: page.image.facebook | default: page.image | prepend: seo_url | escape %}
+  {% assign seo_page_image = page.image.path | default: page.image.facebook |
+default: page.image %}
```
Comment (benbalter) [inline lib/template.html]: @pathawks is this line break intentional? Does it matter?

Comment (pathawks) [inline lib/template.html]: In the process of changing text editors; didn't notice it was hard wrapping this :rage2:

Comment (pathawks) [inline lib/template.html]: Fixed via #79

Comment (benbalter) [inline lib/template.html]: > In the process of changing text editors

Hopefully to Atom! 😄

Comment (pathawks) [inline lib/template.html]: > Hopefully to Atom

The other direction, actually 😞

**Pull request merged**